general-information.md

```diff
@@ -73,14 +74,32 @@ Additionally, `device_id` must remain constant for the device's lifetime of serv
 
 References to geographic datatypes (Point, MultiPolygon, etc.) imply coordinates encoded in the [WGS 84 (EPSG:4326)][wgs84] standard GPS or GNSS projection expressed as [Decimal Degrees][decimal-degrees]. When points are used, you may assume a 20 meter buffer around the point when needed.
 
-Whenever an individual location coordinate measurement is presented, it must be
-represented as a GeoJSON [`Feature`][geojson-feature] object with a corresponding [`timestamp`][ts] property and [`Point`][geojson-point] geometry:
+### Geographic Telemetry Data
+
+Whenever a vehicle location coordinate measurement is presented, it must be
+represented as a GeoJSON [`Feature`][geojson-feature] object with a corresponding
+`properties` object with the following properties:
+
+
+| Field          | Type           | Required/Optional     | Field Description                                            |
```

Comment:
Is it worth adding horizontal accuracy and vertical accuracy as required if available? Would address #73

Reply:
I added `accuracy` because that's what is present in Agency. I'd be happy to add horizontalAccuracy and verticalAccuracy but maybe it warrants a bit of discussion on:
- Are providers able to report on horizontal and vertical accuracy ? AFAIK most GPS chipsets report these values but I don't know whether operators that support `accuracy` today are simply reporting horizontal accuracy or something else…
- Should we make a corresponding (and potentially breaking) change on Agency for consistency?